### PR TITLE
Add descriptions to schema #1

### PR DESCRIPTION
**Pull request by edwardmedia**
Part of https://github.com/terraform-providers/terraform-provider-google/issues/6105

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

**Comment by ndmckinley** [comment]
Would you mind collapsing that history a bit?  It seems something has gone a little wrong there.

File: third_party/terraform/resources/resource_bigtable_instance.go [hunk 1]
```diff
@@ -68,27 +71,32 @@ func resourceBigtableInstance() *schema.Resource {
...
 
 			"instance_type": {
 				Type:         schema.TypeString,
 				Optional:     true,
 				Default:      "PRODUCTION",
 				ValidateFunc: validation.StringInSlice([]string{"DEVELOPMENT", "PRODUCTION"}, false),
+				Description: `The instance type to create. One of "DEVELOPMENT" or "PRODUCTION". Defaults to "PRODUCTION".`,
 			},
 
 			"deletion_protection": {
```
Comment (danawillow) [inline third_party/terraform/resources/resource_bigtable_instance.go:102]: This is missing one

File: third_party/terraform/resources/resource_bigtable_instance.go [hunk 2]
```diff
@@ -68,27 +71,32 @@ func resourceBigtableInstance() *schema.Resource {
...
 						"storage_type": {
 							Type:         schema.TypeString,
 							Optional:     true,
 							Default:      "SSD",
 							ValidateFunc: validation.StringInSlice([]string{"SSD", "HDD"}, false),
+							Description: `The storage type to use. One of "SSD" or "HDD". Defaults to "SSD".`,
 						},
 					},
 				},
+				Description: `A block of cluster configuration options. This can be specified 1 or 2 times. See structure below.`,
```
Comment (danawillow) [inline third_party/terraform/resources/resource_bigtable_instance.go:85]: Can you move this to where the other properties for `cluster` are defined? It's a bit hard to find all the way down here.

Comment (danawillow) [reply third_party/terraform/resources/resource_bigtable_instance.go:85]: Also, you can remove "See structure below"

**Pull request closed without merging**